src/main/java/com/gluonhq/richtext/viewmodel/RichTextAreaViewModel.java

```diff
@@ -229,4 +240,49 @@ public void undo() {
         this.textBuffer.undo();
     }
 
+    private void previousWord() {
+        int textLength = getTextLength();
+        if (textLength <= 0) {
+            return;
+        }
+        if (wordIterator == null) {
+            wordIterator = BreakIterator.getWordInstance();
+        }
+        String text = textBuffer.getText();
```

Comment:
That comment is for an outdated version:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/2043230/154715367-2c8c874e-92c7-4e96-b76e-1c3c980eb52a.png">

Precisely to avoid it we added `PieceCharacterIterator`